Framework/Source/Graphics/Model/Loaders/AssimpModelImporter.cpp

```diff
@@ -648,7 +662,9 @@ namespace Falcor
     {
         initializeBones(pScene);
 
-        if (pScene->HasAnimations())
+        // Create animation controller as long as there are bones.
+        // This will render bind pose if there are no animations.
```

Comment:
Can you open an issue for this? We should (a) detect when the bones were generated by ASSIMP, and if that's the case (b) bake the bind-pose

Reply:
👍